### PR TITLE
Added new commands for okteto namespace - sleep and wake

### PR DESCRIPTION
**Pull request by NitishKumar06**
Fixes: #3470

**Note:** This is a WIP PR - the changes applied are running perfectly. However, I believe that code can be improved considering the file _delete.go_. The test cases need to be added as well. I'm raising this PR before hand to make sure we don't keep bunch of suggestions at the end to be implemented all at once rather it's better if suggestions are implemented side by side.

### CODE REVIEW

**Comment by jLopezbarb** [comment]
Code looks good. I tested the branch and everything seems correct. I'll leave it open to see if anyone wants to review it but LGTM

**Comment by NitishKumar06** [comment]
Squashed the commits

**Comment by NitishKumar06** [comment]
> Thanks @NitishKumar06. The docs should be updated in the https://github.com/okteto/docs. You can create the PR to gather feedback

Yes, I'm aware of it. Just mentioning this here so that anyone who takes a look at this PR and would like to proceed with providing docs can go ahead, otherwise I'd love to do the same.

File: cmd/namespace/wake.go
Comment (jLopezbarb) [inline cmd/namespace/wake.go:34]:
```suggestion
			if err := contextCMD.NewContextCommand().Run(ctx, &contextCMD.ContextOptions{Namespace: nsToWake, Show: true}); err != nil {
```

File: cmd/namespace/sleep.go
Comment (jLopezbarb) [inline cmd/namespace/sleep.go:35]:
```suggestion
			if err := contextCMD.NewContextCommand().Run(ctx, &contextCMD.ContextOptions{Namespace: nsToSleep, Show: true}); err != nil {
```

**Review by jLopezbarb — CHANGES_REQUESTED** [review]

File: cmd/namespace/wake.go
Comment (NitishKumar06) [reply cmd/namespace/wake.go]: marking as Resolved!

**Review by jLopezbarb — APPROVED** [review]

**Comment by NitishKumar06** [comment]
Ping @jLopezbarb

**Comment by jLopezbarb** [comment]
I'm waiting for the next release to be cut to merge this PR. I already tested it and it works perfect.

**Comment by NitishKumar06** [comment]
When is the next release?

**Comment by jLopezbarb** [comment]
The plan is that every month we do a CLI release, but first we need to test that everything is fine and we do a cut creating a beta release. We'll be ready to cut the beta between today and tomorrow, and after that we can merge this PR

**Comment by NitishKumar06** [comment]
> The plan is that every month we do a CLI release, but first we need to test that everything is fine and we do a cut creating a beta release. We'll be ready to cut the beta between today and tomorrow, and after that we can merge this PR

This is amazing! Do you have any suggestions how can I contribute more closely for the next release apart from solving issues?

**Comment by jLopezbarb** [comment]
> > The plan is that every month we do a CLI release, but first we need to test that everything is fine and we do a cut creating a beta release. We'll be ready to cut the beta between today and tomorrow, and after that we can merge this PR
> 
> This is amazing! Do you have any suggestions how can I contribute more closely for the next release apart from solving issues?

We release a beta one week before doing the official release. We test on that beta that everything is working smoothly. I'll talk with the team to create open that process to the community 😄

**Comment by jLopezbarb** [comment]
This PR is a game change, at least for me. I'm very glad to merge this feature. Thanks a lot again @NitishKumar06

**Pull request merged**
